events/wmt20.md

```diff
@@ -217,3 +217,59 @@ Friday, November 20, 2020
 | | [***The Tatoeba Translation Challenge – Realistic Data Sets for Low Resource and Multilingual MT***](https://aclanthology.org/2020.wmt-1.139.pdf) <br>Jörg Tiedemann |
 | | [***Human-Paraphrased References Improve Neural Machine Translation***](https://aclanthology.org/2020.wmt-1.140.pdf) <br>Markus Freitag, George Foster, David Grangier and Colin Cherry |
 | |	[***Incorporating Terminology Constraints in Automatic Post-Editing***](https://aclanthology.org/2020.wmt-1.141.pdf) <br>David Wan, Chris Kedzie, Faisal Ladhak, Marine Carpuat and Kathleen McKeown |
+
+## Results
+
+Full results of the shared tasks: [*Findings of the 2020 Conference on Machine Translation (WMT20)*](https://aclanthology.org/2020.wmt-1.1.pdf)
+
+### News translation
+
+#### → English
+Human evaluation based on segment ratings with document context.
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| Chinese → | 77.5 | 0.102 | `VolcTrans` |
+| Czech → | 78.3 | 0.118 | `CUNI-DocTransformer` |
+| German → | 82.6 | 0.228 | `VolcTrans` |
+| Japanese → | 75.1 | 0.184 | `Tohoku-AIP-NTT` |
+| Polish → | 77.2 | 0.131 | `SRPOL` |
+| Russian → | 79.3 | 0.124 | `Online-G` |
+| Tamil → | 68.7 | 0.203 | `GTCOM` |
+
+Human evaluation based on segment ratings without document context.
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| Inuktitut → | 73.1 | 0.168 | `NiuTrans` |
+| Khmer → | 69.0 | 0.168 | `Online-B` |
+| Pashto → | 67.3 | 0.032 | `Online-B` |
+
+#### English →
+Human evaluation based on segment ratings with document context.
+
+| Language pair | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- |
+| → Chinese | 80.6 | 0.568 | `HUMAN-B` |
```

Comment:
I guess this was a human, not system?  Just for comparison's sake

So we should always show the top performing system.

If we want to show how well humans did, we can add another column or two (for best and worst).